openc3/lib/openc3/models/target_model.rb

```diff
@@ -730,7 +730,6 @@ def deploy_microservices(gem_path, variables, system)
             ["CYCLE_SIZE", @cmd_log_cycle_size]
           ],
           topics: command_topic_list,
-          target_names: [@name],
```

Comment:
Why did these all go away?

Reply:
Because they aren't needed for the Logging Microservices.  This will prevent creating a System in those processes and save a lot of RAM.